src/App.js

```diff
@@ -74,26 +67,22 @@ function App() {
       }
     }
 
-    gameInfo = await getQuestionInfo(playerName);
-    points.innerText = gameInfo.score;
+    gameInfo = await nextGameInfo;
+    points.innerText = getScore(playerName);
     setTimeout(() => {
       generateGameField();
     }, 2500);
   }
```

Comment:
we forgot one line of code
```suggestion
    gameInfo = await nextGameInfo;
    points.innerText = getScore(playerName);
    nextGameInfo = getQuestionInfo(playerName);
    setTimeout(() => {
      generateGameField();
    }, 2500);
  }
```